docs-preview/.vscodeignore

```diff
@@ -1,11 +1,15 @@
-.vscode/**
-.vscode-test/**
-out/test/**
-test/**
-src/**
-**/*.map
+.vscode
+.vscode-test/
+node_modules
+out/
+src/
 .gitignore
 tsconfig.json
 .markdocs/**
 install.Lock
-install.Begin
\ No newline at end of file
+install.Begin
+webpack.config.js
+images/howto.gif
```

Comment:
Hey @bharney0 , can we wildcard the images directory so we don't have to add the individual files?

Reply:
We actually need one of the image files for the images/docs-logo-ms.png. Wasnt sure how to exclude everything except the logo

Reply:
Ah, got it.